pinc/charsuite-medievalist-supplement.inc

```diff
@@ -0,0 +1,65 @@
+<?php
+include_once($relPath."CharSuites.inc");
+
+$charsuite = new CharSuite("medievalist-supplement", _("Medievalist supplement"));
+$charsuite->description = _("Characters used for transcription and analysis of medieval writings");
+$charsuite->codepoints = [
+    # https://www.pgdp.net/wiki/User:Bunny-crunch/Old_orthographies_character_suite
+    'U+0100-U+0103',
+    'U+0111-U+0115',
+    'U+0118-U+0119',
+    'U+0127',
+    'U+012a-U+012d',
+    'U+014c-U+014f',
+    'U+016a-U+016d',
+    'U+017F',
+    'U+0180',
+    'U+01bf',
+    'U+01e2',
+    'U+01e3',
+    'U+01ea-U+01eb',
+    'U+01f7',
+    'U+01fc-U+01fd',
+    'U+021c-U+021d',
+    'U+0232-U+0233',
+    'U+204a',
+    'U+a734-U+a735',
+    'U+a751',
+    'U+a753',
+    'U+a755',
+    'U+a75d',
+    'U+a765',
+    'U+a76b',
+    'U+a76d',
+    'U+a770',
+];
+
+$charsuite->reference_urls = [
+    "https://www.pgdp.net/wiki/User:Bunny-crunch/Old_orthographies_character_suite",
```

Comment:
We should promote this page out of bunny-crunch's userspace if we want to refer to it as a reference URL.

Reply:
Done